src/model/Computation.js

```diff
@@ -1,3 +1,5 @@
+/* global console */
+
```

Comment:
I only add those where my in-editor linter is whining because it makes other issues more visible. Should I remove it?